.github/workflows/e2e_cloud.yml

```diff
@@ -0,0 +1,82 @@
+name: E2E Integrations Test Cloud
+
+concurrency:
+  group: e2e-cloud-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - staging
+      - trying
+  pull_request:
```

Comment:
Since this workflow is already registered, you can delete `on push` and `on pull_request` sections. It will be available for manual run.

Also, need not forget to add periodic run.

Reply:
Im doing same as for fluvio now, by running on an hourly basis

```
on:
  workflow_dispatch:
  schedule:
    - cron: "0 * * * *"
```

Reply:
I would propose running it on daily bases. The activity on this repo is expected to be much less than on Fluvio.